Library/Homebrew/sorbet/files.yaml

```diff
@@ -965,4 +963,4 @@ strict:
   - ./requirements.rb
   - ./requirements/osxfuse_requirement.rb
   - ./rubocops.rb
-  - ./rubocops/rubocop-cask.rb
+  - ./rubocops/rubocop-cask.rb
```

Comment:
@issyl0 Oops!

Reply:
Oh, it's missing a trailing new line.

Reply:
Yup, unfortunately! I've done this a few times myself 😭